ra2ce/analyses/indirect/analyses_indirect.py

```diff
@@ -50,6 +50,86 @@
 from ra2ce.graph.networks_utils import buffer_geometry, graph_to_gdf, graph_to_gpkg
 
 
+def single_link_redundancy(
+        graph: nx.Graph, analysis: AnalysisSectionIndirect
+):
```

Comment:
Why has this been extracted from the container class below?

Reply:
I don't understand this comment.